data_policy.md

```diff
@@ -71,9 +71,13 @@ and the [ISIS Privacy Policy](https://www.isis.stfc.ac.uk/Pages/Privacy.aspx)),
 
 3.4.2 Access to raw data obtained from an experiment is restricted to the experimental team for an embargo period of three years after the end of the experiment. Thereafter, it will become available to the wider community under the [CC-BY license](https://creativecommons.org/licenses/by/4.0/) that obliges users to acknowledge ISIS and cite [Digital Object Identifiers (DOI)](https://www.isis.stfc.ac.uk/Pages/Digital-Object-Identifiers-%28DOIs%29-for-ISIS-Data.aspx) that are linked to ISIS data. Any PI that wishes their data to remain ‘restricted access’ for a longer period will be required to make a special case to the Director of ISIS.
 
-3.4.3 During the proposals process, some elements of the metadata associated with an experiment (such as the title) are indicated as being public; these will be made publicly accessible after the proposal has been accepted. All other elements of the associated metadata are handled in the same way as the raw experimental data in regards to the embargo period.
+3.4.3 ISIS is required to comply with national legislation regarding personal and sensitive data. In such cases, even if the experiment is a result of grant funding, access to this data will remain restricted.
 
-3.4.4 Any PI who wishes their data to be made publicly accessible before the end of the embargo period can ask ISIS to enable this by emailing isisdata@stfc.ac.uk.
+3.4.4 Personal and sensitive data is handled in compliance with all ethical requirements.
+
+3.4.5 During the proposals process, some elements of the metadata associated with an experiment (such as the title) are indicated as being public and may be made publicly accessible, for example when minting a DOI. All other elements of the associated metadata are handled in the same way as the raw experimental data regarding the embargo period.
```

Comment:
Comment from Philip King:

"Apologies, but I’m still not clear regarding the change at 3.4.3.  I’ll describe what happens below, and then you can work out how best to state this within the policy (sorry if I’ve just misunderstood the changes):
 
- Certain elements of a proposal may be made public at any time. These include the proposal title, short abstract, investigators and their institutions, the instrument applied for and the number of days applied for and awarded.
- The elements of a proposal that can be made public are indicated in the proposal system and can be seen by applicants at the time of application.
- The above is true whether or not a proposal is allocated time.
- Three years after the data is taken, the data itself is made available to the wider community. The elements of the original proposal that can be made public will be associated with the raw data, but no further details of the original proposal will be made public with the data – the full proposal is not made public at any point."